api/napi/src/types/image_data.rs

```diff
@@ -22,21 +26,25 @@ impl From<Image> for JsImageData {
 
 #[napi]
 impl JsImageData {
+    /// Constructs a new image with width 0 and height 0.
     #[napi(constructor)]
     pub fn new(width: u32, height: u32) -> Self {
         Self { inner: Image::from_rgba8(SharedPixelBuffer::new(width, height)) }
     }
 
+    /// Returns the width of the image in pixels.
     #[napi(getter)]
     pub fn width(&self) -> u32 {
         self.inner.size().width
     }
 
+    /// Returns the height of the image in pixels.
     #[napi(getter)]
     pub fn height(&self) -> u32 {
         self.inner.size().height
     }
 
+    /// Returns the image as buffer.
```

Comment:
Sadly this is not very useful documentation. In the generated docs there's no link to what "Buffer" is, nor is there any explanation here what it might be. I guess there can't be link since it's a napi internal type.

The napi docs say that their Buffer is a subclass of Uint8Array, so I think that is what the signature in the documentation should show: This property is a UInt8Array.

Reply:
I've added a description.